client/src/features/action/needs/NeedFormGoods.tsx

```diff
@@ -69,6 +70,24 @@ function NeedForm() {
     }));
   };
 
+  const handleSubmit = async (evt: React.FormEvent) => {
+    evt.preventDefault();
+    const res = await fetch('http://localhost:3001/api/assets', {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json',
+      },
+      body: JSON.stringify(formData),
```

Comment:
not required but you could
log in
get the current user from `UserContext` similar to how we do in `client/src/features/users/Inbox/Inbox.tsx`
then spread the user.id into the formData as the `poster_id` property